src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+// 1: отримання елементу з html;
+
+const element = document.querySelectorAll('span.population');
+// 2:перетворення рядка в число
+const allText = Array.from(element).map((el) => {
+  const value = parseInt(el.textContent.replace(/\D/g, ''));
+
+  return isNaN(value) ? 0 : value;
+});
+// 3:знаходжу загальне значення
+
+const total = allText.reduce((sum, num) => sum + num, 0);
+
+// 4: середнє значення
+
+const average = total / allText.length;
```

Comment:
Ensure that `allText.length` is not zero before performing division to calculate the average. If there are no elements, this will result in a division by zero error.